ddtrace/internal/telemetry/writer.py

```diff
@@ -32,25 +32,63 @@ def _get_interval_or_default():
     return float(os.getenv("DD_TELEMETRY_HEARTBEAT_INTERVAL", default=60))
 
 
+class _TelemetryClient:
+    def __init__(self, endpoint, headers):
+        # type: (str, Dict) -> None
+
+        self._agent_url = get_trace_url()
+        self._endpoint = endpoint
+        self._encoder = JSONEncoderV2()
+        self._headers = headers.copy()
+
+    def url(self):
+        return "%s/%s" % (self._agent_url, self._endpoint)
+
+    def send_event(self, request):
+        # type: (Dict) -> Optional[httplib.HTTPResponse]
+        """Sends a telemetry request to the trace agent"""
+        resp = None
+        try:
+            rb_json = self._encoder.encode(request)
```

Comment:
What's the benefit of `JSONEncoderV2()`? should we use this class too in: https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/internal/remoteconfig/client.py#L424

Reply:
Honestly there is no real benefit here aside from aligning with existing writers. This encoder is used in the logs writer to format tracers. Here we only use the `encode` method which simply calls `json.JSONEncoder.encode()` This just call json.